pages/linux/ffuf.md

```diff
@@ -0,0 +1,24 @@
+# ffuf
+
+> A fast web fuzzer written in Go.
+> More information: <https://github.com/ffuf/ffuf>.
+
+- Directory discovery using a wordlist on a target website colorized and verbose:
```

Comment:
What exactly does this search for? Also, example descriptions should always be in imperative form, e.g. "Find directories on a target website ...".

Reply:
addressed in 4fe26e86dfcc04de1115ef613c87693a5e0b7333